Binary files orig/humanleague/src/.NDArray.h.swp and patched/humanleague/src/.NDArray.h.swp differ
diff -Nru orig/humanleague/src/NDArray.h patched/humanleague/src/NDArray.h
--- orig/humanleague/src/NDArray.h	2023-04-17 20:02:36.000000000 +0200
+++ patched/humanleague/src/NDArray.h	2024-02-15 08:29:31.546660700 +0100
@@ -9,6 +9,7 @@
 #include <cstddef>
 #endif
 #include <cassert>
+#include <cstdint>
 
 // The array storage
 template<typename T>
